osquery/sql/sqlite_util.cpp

```diff
@@ -223,6 +223,25 @@ class StringEscaperVisitor : public boost::static_visitor<> {
   }
 };
 
+class SizeVisitor : public boost::static_visitor<> {
+ public:
+  template <typename T>
+  void operator()(T& t) const {
+    size += sizeof(t);
+  }
+
+  uint64_t get_size() const {
+    return size;
+  }
+
+  SizeVisitor() {
+    size = 0;
+  }
+
+ private:
+  mutable uint64_t size;
```

Comment:
mutable? visitor is not cont type.

Reply:
perhaps if `operator()` took a const reference?

Reply:
I believe purpose of mutable specifier is to add an exception, amongst other class members and there is only one member, I dont see any exceptional scenario here.